scripts/client/load_guideline_profile.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import os\n",
+    "import sys\n",
+    "import csv\n",
+    "import django\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "\n",
+    "from pathlib import Path\n",
+    "from django.conf import settings\n",
+    "from django.db import connection\n",
+    "from hawc.apps.vocab.models import GuidelineProfile"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Export spreadsheet with corresponding HAWC vocab ids"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "SELECT\n",
+    "    endpoint_target.id,\n",
+    "    endpoint_category.name AS endpoint_category,\n",
+    "\tendpoint_type.name AS endpoint_type,\n",
+    "    endpoint_target.name AS endpoint_target\n",
+    "FROM\n",
+    "    public.vocab_term AS endpoint_target\n",
+    "LEFT JOIN\n",
+    "    public.vocab_term AS endpoint_type ON endpoint_target.parent_id = endpoint_type.id\n",
+    "LEFT JOIN\n",
+    "    public.vocab_term AS endpoint_category ON endpoint_type.parent_id = endpoint_category.id\n",
+    "WHERE\n",
+    "    endpoint_target.type = 4 AND endpoint_target.namespace=2;\n",
+    "\"\"\""
```

Comment:
This notebook will probably be moved - mainly to demonstrate how guideline profiles were loaded (for review)

To load the model, I updated the endpoint ids in the given guideline_profiles spreadsheet to be the "endpoint target" (effect_subtype) id in the equivalent hawc vocab, and included the SQL logic for getting that here. Hopefully I didn't misunderstand the endpoints/goal here, but def let me know if I did!